tensorflow_addons/layers/normalizations_test.py

```diff
@@ -246,6 +246,81 @@ def test_groupnorm_conv(self):
             model.fit(x=x, y=y, epochs=1)
             self.assertTrue(hasattr(model.layers[0], "gamma"))
 
+    def test_groupnorm_correctness_1d(self):
+        model = tf.keras.models.Sequential()
+        norm = GroupNormalization(input_shape=(10,), groups=2)
+        model.add(norm)
+        model.compile(loss="mse", optimizer="rmsprop")
+
+        x = np.random.normal(loc=5.0, scale=10.0, size=(1000, 10))
+        model.fit(x, x, epochs=5, verbose=0)
+        out = model.predict(x)
+        out -= tf.keras.backend.eval(norm.beta)
+        out /= tf.keras.backend.eval(norm.gamma)
```

Comment:
You've used both `self.evaluate` and `tf.keras.backend.eval`. Stick to the former throughout the PR.

Reply:
Thanks @Squadrick. Do you also suggest using seed for tf.random, like [here](https://github.com/tensorflow/addons/blob/eb77ce281997305703183204fac266f839b7e75f/tensorflow_addons/optimizers/lookahead_test.py#L110)?

Reply:
Yup, that too. Forgot to mention it.